### ISSUE TITLE
VO+Safari does not recognize button elements

### ISSUE THREAD

**Issue by shelbyelavsky** [fizzstudio/svg-aria-test#67]
This is an issue in reference to #23 which is related to the **"primitives and nested role=button"** branch.

<img width="614" alt="Screenshot of VoiceOver in test environment. A blue circle is highlighting Text Viewer text that reads '7  You are currently on a text element'. " src="https://user-images.githubusercontent.com/98043076/161110535-4d831aac-ac7d-4f95-bcbf-4078bcf99d24.png">

### Issue Summary
VoiceOver does not recognize toggleable elements on SVG or HTML as expected. It only recognizes the “3 Text” as a toggle button in SVG, but VoiceOver still reads what should be labels of the element as text (3 before the button and text after.) 

### Video of issue 
Example starts at 00:00:21 and 00:00:39
https://user-images.githubusercontent.com/98043076/161101206-0c294fa7-fa83-40ab-bcc3-42e1db6c43d2.mp4

### Steps to reproduce
Starting at the "Interactions Instructions" heading: Use VoiceOver down arrow key to navigate into the SVG space. Press the down arrow key 10 times to reach the "3 Text toggleable button." 

### Expected behavior (Pass/Fail)
- **FAIL** We expect that a screen reader (SR) can navigate to each element with a role and aria-label using a keyboard (SVG, g, and all SVG primitives).
- **FAIL** We expect that the visual focus indicator moves to the correct element and has the correct bounding dimensions.
- **FAIL** We expect that the visual focus indicator will remain in view at all times and may even adjust the scroll position of the page while navigating.
- **FAIL** The SVG is labelled by the chart's title and subtitle (these should be read as a single text when navigating to the root element).
- **FAIL** The title and subtitle are hidden from a screen reader because they are announced on their parent element instead. These elements should not be read out loud.
- **FAIL** The elements are grouped in pairs. The screen reader should be able to separately navigate to the group before navigating to each pair.
- **FAIL** Each primitive (rect, circle, path, text) should be accessible by a screen reader and read the number and label for that element.
- **FAIL** We expect that a SR will announce the aria-label correctly for each element, followed by its role.
- **FAIL** The aria label should announce the number, followed by the type of primitive (eg "7 Rect"), even though the text may be displayed visually in a different order. This is designed intentionally to ensure that the ARIA is working correctly.
- **FAIL** We expect that different SVG primitives (rect, circle, path, text) will be consistent with each other based on previous expectations.
- **FAIL** We expect that the SVG elements will provide the same experience as the HTML elements (below the SVG space).
- **???** We expect that if the main roles are not used, the fallback roles will be (if they are enabled). We are not sure how to design for this initially, but are mentioning it as important.
- **Pass** We expect that a SR will be able to toggle the element on or off.
- **Pass** We expect that a SR will announce the toggle state correctly when navigated to.
- **Pass** We expect that a SR will announce the toggle state correctly when changed.
- **Pass** We expect that the aria live region for each test type (HTML and SVG) will correctly announce that something has happened with each toggle.

### Technical Details
VoiceOver (Default)
Safari 15.1 (17612.2.9.1.20)
MacBook Pro/ macOS Monterey 12.0.1
